src/coreComponents/physicsSolvers/NonlinearSolverParameters.cpp

```diff
@@ -149,6 +149,36 @@ NonlinearSolverParameters::NonlinearSolverParameters( string const & name,
     setApplyDefaultValue( 0 ).
     setDescription( "Flag to decide whether to iterate between sequentially coupled solvers or not." );
 
+  registerWrapper( viewKeysStruct::useDBCString(), &m_useDBC ).
+    setInputFlag( InputFlags::OPTIONAL ).
+    setApplyDefaultValue( 0 ).
+    setDescription( "Enable Dissipation-based continuation flux" );
+
+  registerWrapper( viewKeysStruct::omegaDBCString(), &m_omegaDBC ).
+    setApplyDefaultValue( 1 ).
+    setInputFlag( InputFlags::OPTIONAL ).
+    setDescription( "Factor by which DBC flux is multiplied" );
+
+  registerWrapper( viewKeysStruct::continuationDBCString(), &m_continuationDBC ).
+    setApplyDefaultValue( 1 ).
+    setInputFlag( InputFlags::OPTIONAL ).
+    setDescription( "Flag for enabling continuation parameter" );
+
+  registerWrapper( viewKeysStruct::miscibleDBCString(), &m_miscibleDBC ).
```

Comment:
Can't we deduce that from the fluid model in use and `m_useDBC` ?

Reply:
good question, @francoishamon can you point me how to do that?